modules/projects/projectList.php

```diff
@@ -8,7 +8,8 @@
     mysqli_query($db, $deleteProject);
 }
 
-function getProjects($db) {
+function getProjects($db,$isFiltered) {
```

Comment:
change $isFiltered to $isFiltered=false

By default it will not be filtered(false), if we want to filter, when calling the function we will have to indicate by writing getProjects($db, $isFiltered=true), otherwise getProjects($db) will suffise